src/web/app/pages-instructor/instructor-search-page/instructor-search-page.component.ts

```diff
@@ -18,6 +16,24 @@ export interface SearchStudentsTable {
   sections: StudentListSectionData[];
 }
 
+/**
+ * Data object for communication with child comment result component
+ */
+export interface SearchCommentsTable {
```

Comment:
The type definition should be in `src/web/app/pages-instructor/instructor-search-page/comment-result-table/comment-result-table.component.ts` right?

Reply:
Then also shift the `SearchStudentsTable` to the respective component too? Sounds better to place it there too

Reply:
Yes. But please do it in a separate PR.